assets/js/admin/course-general-sidebar.js

```diff
@@ -11,9 +11,13 @@ import {
 } from '@wordpress/components';
 import { useEntityProp } from '@wordpress/core-data';
 import apiFetch from '@wordpress/api-fetch';
-import { useState, useEffect } from '@wordpress/element';
+import { useState, useEffect, useMemo } from '@wordpress/element';
 
+/**
+ * Internal dependencies
+ */
 import editorLifecycle from '../../shared/helpers/editor-lifecycle';
+import { applyFilters, doAction } from '@wordpress/hooks';
```

Comment:
I think `doAction` is never used?

Reply:
Right! I am surprised that the JS linting didn't catch this 🤔 

Fixed in c5b4c5494